examples/ex08_materials/include/kernels/ExampleDiffusion.h

```diff
@@ -42,6 +42,6 @@ class ExampleDiffusion : public Diffusion
    * This MooseArray will hold the reference we need to our
```

Comment:
Yep... you're spot on.  There was a time before the `MaterialProperty` Typedef.  This is one reason why more comments are not always better... they tend to last longer than the code.